### DISCUSSION TITLE
One Touch Display stopped working?

### DISCUSSION THREAD

**Discussion by 64Spaces** [aqualinkd/AqualinkD#315] [Q&A] [answered]
Hello, since there are several experts using this plugin, I’m hoping I can ask a question on these Discussion pages. I have been using AqualinkD for several years and have no issues with the plugin. 

However, a week or so ago, my Jandy One Touch Display stopped working. I can still use the AqualinkD webpage and HomeKit to control everything. 

Since AqualinkD still works, is it safe to assume the LCD Display Board is now defective? Or could there be something wrong with the Main Aqualink Power Center Controller? Do these LCD Display Boards have a history of failure? There are a ton of the displays on eBay, priced around $750. 

I’m not sure if I should replace one or the other or I need to replace both the display and power center?

Thanks so much for your help.

**Comment by sfeakes** [comment, marked as answer]
If AqualinkD is running then the control panel is fine.
The Onetouch & PDA remotes to tend to crap out over time, so it's probably just the One Touch remote.  But you should first check the wiring running to the remote.  The 4 wires are ( + / - / Data + / Data - ). Since you have no display it will be a power problem. Check the + and - for power if you have a multimeter / volt meter / test probe. You should have (about)24v DC across the + and - wires.  They are Pin 1 and Pin 4. 
Pin 2 & 3 are Data, don't check those.
As for what those wires & terminals look like, I can show you on the control panel side (also in the wiki), but I don't have a one touch remote, so not sure on that side of things.

Also with the AqualinkD simulators you don't even need a one touch remote. Any programming / setup of the panel can be done through the AqualinkD one touch simulator.  (The new simulators were added on AqualinkD 3.2.4)

**Comment by 64Spaces** [comment]
Great info! Thanks so much!
